lib/resources/r4/communication.yaml

```diff
@@ -189,5 +185,30 @@ fields:
         }
       ]
     }
-  action:
-    - create
+  action: create
+
+- name: Communication Email-Status Extension
+  terminology_name: extension[x].valueCodeableConcept
+  required: 'No'
+  type: CodeableConcept
+  description: Status extension for Communication
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/email-status",
+      "valueCodeableConcept": {
+        "coding": [
+          {
+            "code": "in-progress",
+            "system": "https://www.hl7.org/fhir/codesystem-task-status.html"
+          }
+        ]
+      }
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/email-status?_format=json
+  binding:
+    description: Communication Email Status Extension
+    terminology:
+    - display: Communication Email Status Extension
+      system: https://www.hl7.org/fhir/codesystem-task-status.html
```

Comment:
This is wrong

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/files/4670196dfe7d6acd28005f6e8fe73f0a080d37f3..cbf73631d007bdd6cc4c8f9bdc432cffd7047e54#diff-f08ae1e4c063b9a36a22adaa43fecf21R196